ycoheNvidia/sonic-utilities
# implemented secure upgrade

#### What I did
Implemented secure upgrade in Sonic
#### How I did it
It includes image signing during build (in sonic buildimage repo) and verification during image install (in sonic-utilities).
#### How to verify it
Feature is used to allow image was not modified since built from vendor. During installation, image can be verified with a signature attached to it.
In order for image verification - image must be signed - need to provide signing key and certificate (paths in SECURE_UPGRADE_DEV_SIGNING_KEY and SECURE_UPGRADE_DEV_SIGNING_CERT in rules/config) during build , and during image install, need to enable secure boot flag in bios, and signing_certificate should be available in bios.
#### Previous command output (if the output of a command-line utility has changed)

#### New command output (if the output of a command-line utility has changed)
